src/scss/skin-modern/components/subtitlesettings/_subtitlesettingsclosebutton.scss

```diff
@@ -0,0 +1,28 @@
+.#{$prefix}-ui-subtitlesettingsclosebutton {
```

Comment:
Import variables into file, else the usage of  e.g. `$color-primary` leads to warning in IDEs (`@import '../../variables';`)